ports/geckolib/gecko_bindings/sugar/ns_style_coord.rs

```diff
@@ -3,9 +3,21 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 use bindings::{Gecko_ResetStyleCoord, Gecko_SetStyleCoordCalcValue, Gecko_AddRefCalcArbitraryThread};
+use std::mem;
 use structs::{nsStyleCoord_Calc, nsStyleUnit, nsStyleUnion, nsStyleCoord, nsStyleSides, nsStyleCorners};
 use structs::{nsStyleCoord_CalcValue, nscoord};
 
+impl nsStyleCoord {
+    #[inline]
+    pub fn null() -> Self {
+        // can't construct directly because it has private
+        // fields
+        let mut coord: Self = unsafe { mem::zeroed() };
```

Comment:
/me *mutters something about accessors...*

Is the wrapping of the comment intentionally at ~65 chars?

r=me when it's ready, ideally with the proper wrapping and capitalization in the comment